### PR TITLE
Add page for configuring time zone

### PR DESCRIPTION
**Pull request by kelvinfan001**

### CODE REVIEW

File: modules/ROOT/pages/time-zone.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+----
+variant: fcos
+version: 1.3.0
+storage:
+  links:
+    - path: /etc/localtime
+      target: ../usr/share/zoneinfo/America/New_York
+----
+
+=== Changing the time zone via `timedatectl`
```
[the remark this answers is not in the document]
Comment (lucab) [reply modules/ROOT/pages/time-zone.adoc:44]: I agree. I think this come from the previous Container Linux docs which were wrongly skewed in that direction.
I guess we can merge the interactive list and status actions into a single section, and avoid the setting part.

File: modules/ROOT/pages/time-zone.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+[source,bash]
+----
+$ timedatectl status
+               Local time: Mon 2021-05-17 20:10:20 UTC
+           Universal time: Mon 2021-05-17 20:10:20 UTC
+                 RTC time: Mon 2021-05-17 20:10:20    
+                Time zone: UTC (UTC, +0000)           
+System clock synchronized: yes                        
+              NTP service: active
```
[the remark this answers is not in the document]
Comment (kelvinfan001) [reply modules/ROOT/pages/time-zone.adoc:17]: Not too familiar with `chrony` and NTP, so I just briefly mentioned `chrony` and left a link to the internal docs.

File: modules/ROOT/pages/time-zone.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (kelvinfan001) [reply modules/ROOT/pages/time-zone.adoc:44]: That makes sense, thanks for pointing this out.

File: modules/ROOT/pages/time-zone.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (travier) [reply modules/ROOT/pages/time-zone.adoc:17]: Agree that we should move this part of the internal doc in the official one but that could be another PR.

File: modules/ROOT/pages/time-zone.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Africa/Accra
+Africa/Addis_Ababa
+…
+----
+
+See the https://www.freedesktop.org/software/systemd/man/timedatectl.html[manual page] for more information about how `timedatectl` can be used; however, we do not recommend changing the time zone per-machine imperatively via SSH.
+
+=== Recommended time zone: Coordinated Universal Time (UTC)
+
+To avoid time zone confusion and the complexities of adjusting clocks for daylight saving time (or not) in accordance with regional custom, we recommend that all machines in Fedora CoreOS clusters use UTC. Setting all machines to a single time zone will also help with correlating log files across systems. UTC is the default time zone.
```
Comment (travier) [inline modules/ROOT/pages/time-zone.adoc:36]: I think we should word that in a stronger fashion but I don't have a good suggestion right now.

File: modules/ROOT/pages/time-zone.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+=== Setting the time zone via Ignition
+
+If your applications require a different system time zone, you can do so by setting the local time zone configuration file, https://www.freedesktop.org/software/systemd/man/localtime.html[`/etc/localtime`], to be an absolute or relative symlink to a `tzfile` entry under `/usr/share/zoneinfo/`.
```
Comment (travier) [inline modules/ROOT/pages/time-zone.adoc:40]: In most cases, you can force a timezone for just some application via environment variables: https://wiki.archlinux.org/title/System_time#Per-user/session_or_temporary_settings

**Comment by travier** [comment]
This looks great. My comments are just here to make sure we scare people enough to not change the timezone 🙂

File: modules/ROOT/pages/time-zone.adoc (hunk 3 above)
Comment (kelvinfan001) [reply modules/ROOT/pages/time-zone.adoc]: I've added a couple more reasons not to use a non-UTC time zone :)

**Review by travier — APPROVED** [review]
Looks great. Like the new wording!

**Pull request merged**